generators/client/templates/src/main/webapp/app/blocks/handlers/_state.handler.js

```diff
@@ -62,15 +59,6 @@
                     stateChangeSuccess();
                 }
             });
-
-            function back() {
```

Comment:
Ok then
On 17 Apr 2016 02:04, "Stephen Doxsee" <notifications@github.com> wrote:

> In
> generators/client/templates/src/main/webapp/app/blocks/handlers/_state.handler.js
> <https://github.com/jhipster/generator-jhipster/pull/3397#discussion_r59971330>
> :
>
> > @@ -62,15 +59,6 @@
> >                      stateChangeSuccess();
> >                  }
> >              });
> > -
> > -            function back() {
>
> Not anymore. Even before my PR, this was orphaned code. See how where it
> is still in use in 2.27.2:
>
>    -
>    https://github.com/jhipster/generator-jhipster/blob/v2.x_maintenance/app/templates/src/main/webapp/scripts/app/_app.js#L64
>    -
>    https://github.com/jhipster/generator-jhipster/blob/v2.x_maintenance/app/templates/src/main/webapp/scripts/app/account/login/_login.controller.js#L21
>
> The intention for it seems to have been that if a previousState was set
> due to unauthorized access of a page, the login screen would be shown,
> followed by a "back()" to restore the previousState. But, there is nothing
> calling this "back()" function in 3.0 since it is being handled in the
> login callback.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/jhipster/generator-jhipster/pull/3397/files/41ed3ce95fcc153168d1e838afb0c6d865dafa3b#r59971330>
>